table/properties.go

```diff
@@ -25,10 +25,24 @@ const (
 	ObjectStoreEnabledKey                   = "write.object-storage.enabled"
 	ObjectStoreEnabledDefault               = false
 
-	DefaultNameMappingKey = "schema.name-mapping.default"
-
+	DefaultNameMappingKey       = "schema.name-mapping.default"
```

Comment:
nit: I think it was better before :)

Reply:
keeping the empty space between them instead of having it part of the same group? sure I can put it back :)